spring-cloud-gcp-samples/spring-cloud-gcp-trace-sample/src/test/java/com/example/TraceSampleApplicationIntegrationTests.java

```diff
@@ -82,17 +95,49 @@
 
   private TraceServiceBlockingStub traceServiceStub;
 
-  @BeforeClass
-  public static void checkToRun() {
-    assumeThat(
-        "Google Cloud Trace integration tests are disabled. "
-            + "Please use '-Dit.trace=true' to enable them. ",
-        System.getProperty("it.trace"),
-        is("true"));
+  private static final String SAMPLE_TOPIC = "traceTopic";
+
+  private static final String SAMPLE_SUBSCRIPTION = "traceSubscription";
+
+  static final String projectName =
+      ProjectName.of(ServiceOptions.getDefaultProjectId()).getProject();
+
+  private static TopicAdminClient topicAdminClient;
+
+  private static SubscriptionAdminClient subscriptionAdminClient;
+
+  @BeforeAll
+  static void setup() throws IOException {
+
+    topicAdminClient = TopicAdminClient.create();
+    subscriptionAdminClient = SubscriptionAdminClient.create();
+
+    TopicName topic = TopicName.ofProjectTopicName(projectName, SAMPLE_TOPIC);
+    Topic response = topicAdminClient.createTopic(topic);
+    SubscriptionName subName = SubscriptionName.of(projectName, SAMPLE_SUBSCRIPTION);
+    PushConfig pushConfig = PushConfig.newBuilder().build();
+    int ackDeadlineSeconds = 599;
+    Subscription subResponse =
```

Comment:
Try using `PubSubAdmin` for creating and deleting Pub/Sub resources. Check the documentation to see how usage differs from client library.

Reply:
On it.